bindings/nodejs/lib/wallet/account.ts

```diff
@@ -116,6 +116,20 @@ export class Account {
         return Output.parse(JSON.parse(response).payload) as NftOutput;
     }
 
+    /**
+     * A generic `burn()` function that can be used to prepare to burn native tokens, nfts, foundries and aliases.
+     * @param burn The outputs to burn
+     * @param transactionOptions The options to define a `RemainderValueStrategy`
```

Comment:
I know it was probably already there but I wouldn't mention specific options, sounds unmaintainable.